lib/vagrant-libvirt/config.rb

```diff
@@ -279,26 +279,26 @@ def _get_cdrom_dev(cdroms)
       end
 
```

Comment:
Merge behavior isn't wanted for numa_nodes. Override should happen if numa_nodes is defined in different files and blocks.